### PR TITLE
Fixed Wilks

### PR DESCRIPTION
**Pull request by regislebrun**
The initializtion of the sequential search for the sample size was done using an asymptotic approximation not valid for low quantile levels. It fixes issue #1818.

### CODE REVIEW

File: lib/src/Uncertainty/Algorithm/Simulation/Wilks.cxx [hunk 1]
```diff
@@ -86,6 +96,7 @@ Point Wilks::computeQuantileBound(const Scalar quantileLevel,
 {
   // Compute the needed sample size
   const UnsignedInteger size = ComputeSampleSize(quantileLevel, confidenceLevel, marginIndex);
+  std::cerr << "quantileLevel=" << quantileLevel << ", confidenceLevel=" << confidenceLevel << ", marginIndex=" << marginIndex << ", size=" << size << std::endl;
```
Comment (jschueller) [inline lib/src/Uncertainty/Algorithm/Simulation/Wilks.cxx:99]: drop the cerr

Comment (regislebrun) [reply lib/src/Uncertainty/Algorithm/Simulation/Wilks.cxx:99]: I am rewriting the code, I found a numerical instability thanks to the plot_quantile_estimation_wilks.py example.

Comment (jschueller) [reply lib/src/Uncertainty/Algorithm/Simulation/Wilks.cxx:99]: ah ok, sorry

File: lib/src/Uncertainty/Algorithm/Simulation/Wilks.cxx [hunk 2]
```diff
@@ -45,38 +48,120 @@ Wilks::Wilks(const RandomVector & vector)
...
+  {
+    return 1;
+  }
+
+  UnsignedInteger getOutputDimension() const
+  {
+    return 1;
+  }
+
+  Description getInputDescription() const
```
Comment (jschueller) [inline lib/src/Uncertainty/Algorithm/Simulation/Wilks.cxx:86]: description and str converter methods are useless if the functions is local

File: python/test/t_Wilks_std.py [hunk 3]
```diff
@@ -5,20 +5,11 @@
...
-    import sys
-    print("t_Wilks_std.py", sys.exc_info()[0], sys.exc_info()[1])
+alpha = [0.05 * i for i in range(1, 20)]
+beta = [0.05 * i for i in range(1, 20)]
+for a in alpha:
+    for b in beta:
+        print("alpha=%.2f" % a, "beta=%.2f" % b)
+        for i in range(5):
+            print("n(%.2f" % a, ",%.2f" %b, ",", i, ")=",
+                  Wilks.ComputeSampleSize(a, b, i))
```
Comment (jschueller) [inline python/test/t_Wilks_std.py:15]: seems it fails on win32

Comment (regislebrun) [reply python/test/t_Wilks_std.py:15]: They were there for debugging purpose. They helped me during the development, maybe they would ease to catch future bugs ;-)!

Comment (jschueller) [reply python/test/t_Wilks_std.py:15]: I mean the code of the test is good but they fail to pass on win32

**Pull request merged**